### PR TITLE
CS wrapping

### PR DESCRIPTION
**Pull request by mathstuf**
---
@yumin @vibraphone

### CODE REVIEW

**Comment by mathstuf** [comment]
ParaView doesn't have the bits merged for this yet. One question is about the Remus support. The Meshing module doesn't exist without Remus. Does this make sense?

**Comment by yumin** [comment]
@robertmaynard Rob will be the best person to answer the Remus/Meshing question.

**Comment by robertmaynard** [comment]
I think having the meshing component require remus is reasonable.

File: smtk/bridge/discrete/extension/reader/module.cmake [hunk 1]
```diff
@@ -16,5 +16,6 @@ vtk_module(vtkSMTKDiscreteExt
     vtkRenderingVolume${VTK_RENDERING_BACKEND}
     vtkRenderingContext${VTK_RENDERING_BACKEND}
     vtksys
+    vtkSMTKDiscreteMeshingExt
```
Comment (mathstuf) [inline smtk/bridge/discrete/extension/reader/module.cmake]: This should be conditional based on Remus.

**Comment by mathstuf** [comment]
This should be ready now that ParaView is updated in the superbuild. Any issues with it?

**Comment by yumin** [comment]
+1

**Pull request merged**